src/libs/filter-utils.ts

```diff
@@ -73,4 +114,18 @@ export namespace FilterUtilities {
     if (taskId && task?.toString() !== taskId || !task) return false;
     return true;
   }
+
+  /**
+   * Compares severa phase id to targetPhase id
+   * 
+   * @param severaPhaseId Phase id from Severa
+   * @param targetPhaseId Phase id to compare
+   * @returns 
+   */
+  export const filterByPhaseSevera = (severaPhaseId: string, targetPhaseId: string): boolean => {
```

Comment:
Similar comments here as for the other filter funcitons, fix Docs, return typing and we can figure out if this filtering is necessary if we can filter the data using severaApi params